### ISSUE TITLE
Games with video or sound problems

### ISSUE THREAD

**Issue by lilalien** [CollectorVision/Phoenix-Colecovision#8]
So I've noticed several games with sound or video issues while spending a few hours with the Phoenix tonight. I really had a blast, but I'd love to get these issues rectified. I'll try to update with more as I find them...

Sound issues I've noticed so far:
Mouse Trap (there are two sound effects while playing the game that does not sound the same as the original - think its when you get the bones and when you press 5 to turn into a red dog).
Smurf (the jumping sound in the green valley screens is very off).
Frogger - Some bad sound effects and some video artifacts.
Venture - Sound issue when killing an enemy.

Video issues:
Also, Zaxxon completely froze after existing the first area and the flying airplanes appeared. It locked up hard and required a reset. Also bad artifacts like force fields appearing where they don’t belong and an airplane that just stays on the screen in areas it doesn’t belong.

River Raid has bad video problems too.

I'll post more as I notice them, I'll bring a pen up with me to the game room next time I'm in there and write down more specific sound issues...

**Comment by lilalien** [comment]
Updated first post.. I’ll try to upload a few YouTube videos for the bugs too and link them when done.

**Comment by lilalien** [comment]
Here are some video examples to isolate these bugs I'm experiencing...

**UPDATE: All bugs below occur via AtariMax Ultimate SD multicart, and included Phoenix SD card (via two different set of rom images), and the physical cartridges themselves! So its definitely a core or hardware batch issue!**

River Raid - https://youtu.be/krIamI_0oz8
Smurfs - Rescue at Gargamel's Castle: https://youtu.be/Gt_21ZxOUwc
Venture - https://youtu.be/e5Ei8M5jCRA
Mousetrap - https://youtu.be/Ff-CthbCY1g
Cosmic Avenger - https://youtu.be/mp4YRqHJZNs
Zaxxon 1 - https://youtu.be/qnV_pLR_GU8
Zaxxon 2 - https://youtu.be/CSaMTIsuhUM

**Comment by lilalien** [comment]
Frogger: https://youtu.be/t-OoIBk7fZ0

**Comment by ikrananka** [comment]
I tested all of those games with my Phoenix and AtariMax and do not experience any of the sounds issues that you do.  Might be worth noting here for CollectorVision which firmware version you have in your AtariMax.

When testing the games you listed I did experience major problems with River Raid.  When playing it via the AtariMax it kept randomly resetting the game every few minutes.  At one point the game froze entirely and the graphics on screen became completely garbled.

I tried the original cart and didn't have any issues so far in my testing.

**Comment by ikrananka** [comment]
Just tried River Raid with the AtariMax again and had no issues at all this time.  Weird.

**Comment by lilalien** [comment]
I'll double check my Atarimax firmware, and obtain the latest one to see if that fixes it many of the issues. Apparently I have to e-mail to get that...

I'll update once done.

**Comment by lilalien** [comment]
Apparently my versions are several versions behind and quite old.. I e-mailed to obtain the latest firmware. I do have the files now from elsewhere on the forums, but want to be sure they're the right ones before I proceed.

**Comment by lilalien** [comment]
So just thinking, even if I update the Atarimax multicart firmware and the menu issues go away, it still sounds like the sound/video problems I'm experiencing IN GAME will still be there as they still occur when I drop those same games onto the included Phoenix memory card...

**Comment by ikrananka** [comment]
Also, try using the rom files from my [CV Rom Update Project](https://atariage.com/forums/topic/228832-cv-rom-update-project-commercial-releases-v12/).  These are known good roms and will eliminate the possibility that your problems are related to bad roms.

**Comment by lilalien** [comment]
Hey,

I did some more digging tonight and have some updates... The drama is getting thick!

So I took the games that I was having issue with from my Atarimax, and dropped them on the included memory card and the sound problem was still there. I also picked up the whole system and dragged it to the master bedroom Samsung LED and the same exact sound issues occur there too.

I thought I'd isolate further, and swap out the HDMI cable and that also changed nothing with the bad sound issues.

My roms should be fine, I got them off of known good collections and have been using them for a long time. They work 100% on my real Colecovision hardware, and within emulators but I'll certainly grab yours for kicks to see what happens!

As an example, the bad sound effects in Smurf make the music cut by half in volume and the jumping sound will play distorted and sounds terrible. Then after that sound effect plays, the music returns 1-2 seconds later at normal volume.

I've isolated this problem 100% to either the ROM or the Phoenix itself. It cannot be the power supply, HDMI cable, included memory card, or the Atarimax.

**Comment by lilalien** [comment]
> Also, try using the rom files from my [CV Rom Update Project](https://atariage.com/forums/topic/228832-cv-rom-update-project-commercial-releases-v12/). These are known good roms and will eliminate the possibility that your problems are related to bad roms.

Hey,

So for kicks I replaced the roms that I was having issues with, with the ones from your project. All have the exact same sound issues, however so far so good on the video issues I was experiencing with Zaxxon. I'll play a longer game tomorrow when I'm not about to pass out!

**Comment by lilalien** [comment]
OK, I also tested the physical game cartridges as well and the same issues occur. So its either core related or a bad batch of Phoenix units went out.

I'll post any other sound issues I have with other games as I play with the unit further.

I look forward to a resolution of this, because otherwise its awesome.

**Comment by lilalien** [comment]
Today I replaced the firmware that you shipped with my unit (that I received on Nov 4), with the one in the first post of the Atari Age release thread and have some news to report. It was a core/firmware bug!

So every sound issue mentioned in this thread, in addition to visual bugs have completely gone away. No weird sound effects, no weirdness in Zaxxon, no oddness in River Raid, no jumping sound effect issue with Smurf, no sound effect issue in Venture nor Mouse Trap, music on Cosmic Avenger is great too! All is now working well with that firmware from that thread. Not sure what was wrong with my firmware that came with my unit but I thought you'd want to know the outcome of this!

I did notice one other audio bug however:

Music playing in games such as Donkey Kong (level 2), and Smurf - Rescue in Gargamel's Castle (first area) begins sort of low and gradually changes volume level while you are playing.

**Comment by ikrananka** [comment]
I just tested mine and don't hear the music volume issues you're experiencing.

**Comment by lilalien** [comment]
Very strange...

I'll try my physical cartridges tomorrow morning to see if I get that music volume issue there too... Its like if I make a sound effect the music goes down a little at times as well, then raises back up at some point.

**Comment by lilalien** [comment]
OK, I found the fix for this. I decided to take a look at my TV settings for kicks and I had a setting for SRS TruSound which apparently attempts to normalize the balance of sound! I tried the Phoenix in an input I normally don't use and so I never customized that input to turn off all the junk I don't use for gaming.. Yay!

So I guess we can officially call this closed, and open a new one should new game issues occur?
